### ISSUE TITLE
.git/ directory is published to registry

### ISSUE THREAD

**Issue by thathert** [mauriciopoppe/function-plot#152]
Hey thank you for the great library! I've been running into an issue where npm seems pretty unhappy after installing it though:

```
npm ERR! code EISGIT
npm ERR! path ./node_modules/function-plot
npm ERR! git ./node_modules/function-plot: Appears to be a git repo or submodule.
npm ERR! git     ./node_modules/function-plot
npm ERR! git Refusing to remove it. Update manually,
npm ERR! git or move it out of the way first.
```

Things are fine again if I `rm -fr ./node_modules/function-plot/.git` 

Steps to reproduce:

1. `mkdir testdir && cd $_`
2. `npm init`
3. `npm install --save function-plot`
4. `npm install`

**Comment by mauriciopoppe** [comment]
Thanks for the report, this was recently fixed in v1.19.2, please verify that you have the latest version

**Comment by thathert** [comment]
Ah, that does the trick! I was on 1.19.1 -- thanks!
